Configure

```diff
@@ -1790,8 +1791,16 @@ while (<IN>)
 	}
 close(IN);
 close(OUT);
-rename($Makefile,"$Makefile.bak") || die "unable to rename $Makefile\n" if -e $Makefile;
-rename("$Makefile.new",$Makefile) || die "unable to rename $Makefile.new\n";
+if ((compare($Makefile, "$Makefile.new"))
+	or file_newer('Configure', $Makefile)
+	or file_newer('config', $Makefile)
+	or file_newer('Makefile.org', $Makefile))
```

Comment:
Because otherwise make will fail.

See the following rule in Makefile:

```
Makefile: Makefile.org Configure config
        @echo "Makefile is older than Makefile.org, Configure or config."
        @echo "Reconfigure the source tree (via './config' or 'perl Configure'), please."
        @false

```